src/lib/zf/tcp.c

```diff
@@ -64,6 +64,7 @@ static int __zft_handle_free(zf_stack* stack, zf_tcp* tcp)
 
   zf_stack_tcp_to_res(stack, tcp, &rx_res);
   zfrr_release_port(stack, rx_res);
+  memset(&tcp->laddr, 0, sizeof(tcp->laddr));
```

Comment:
I think this might be superfluous if we're also choosing to also zero this in `zf_tcp_release` as we call that function at the bottom of this function (with no chance to exit early without calling it). Given that it's just some extra work at free time though, I don't think it should matter much if you don't want to change it.

Reply:
Good point!  I won't update now as it has been merged but agree it is probably superfluous.